src/v/serde/parquet/tests/generate_file.cc

```diff
@@ -322,6 +322,7 @@ ss::future<iobuf> serialize_testcase(size_t test_case) {
         writer w(
           {
             .schema = dremel_paper_schema(),
+            .compress = test_case % 2 == 0,
```

Comment:
This is always 0 in this if block?

Reply:
oof I meant to do this for the randomized data only

Reply:
Fixed - thanks!